src/WebJobs.Script.WebHost/Diagnostics/SystemTraceWriter.cs

```diff
@@ -53,6 +54,11 @@ public override void Trace(TraceEvent traceEvent)
                     functionName = value.ToString();
                 }
 
+                if (traceEvent.Properties.TryGetValue(ScriptConstants.TracePropertyHostVersionKey, out value) && value != null)
```

Comment:
Added host version to the event properties when the host starts in [ScriptHostManager.cs](https://github.com/pragnagopa/azure-webjobs-sdk-script/blob/hostVersion/src/WebJobs.Script/Host/ScriptHostManager.cs#L96) and looking up this property when writing logs. Let me know if you think using ScriptHost.Version is better